app/views/posts/show.html.erb

```diff
@@ -1 +1,15 @@
-<h1>Post details</h1>
\ No newline at end of file
+<h1>Post details</h1>
+
+<div class="border h-50 w-50 p-2 m-2 border-dark">
+    <%= @post.title %> <br>  
+    <%= @post.text %>  
+</div> <br>
+<div class="card mb-3 border-secondary" style="width: 540px;">
+    <div >
+      <% @comments.each do |comment| %>
+          <p ><%= comment.author.name %>: <%= comment.text %></p>
+              <% end %>
+    </div>
+      </div>
+
+</div>
```

Comment:
- [ ] Once again, please consider following the given wireframe and adding the missing counters and attributes! 🤓 🔧 

|Template|Your design|
|---|---|
|![image](https://user-images.githubusercontent.com/77069011/222305524-663f4d3e-911f-4f2d-a73b-37331261d545.png) | ![image](https://user-images.githubusercontent.com/77069011/222305560-17ec3d57-3b06-4491-83de-78f76f08468a.png) |

|